utils/http.ts

```diff
@@ -76,13 +76,16 @@ export interface HttpClientOptions {
   headers?: Headers;
   processHeaders?: (headers?: Headers) => Headers | undefined;
   fetcher?: typeof fetch;
+  // Keep empty path segments in the URL
+  keepEmptyPath?: boolean;
```

Comment:
keepTrailingSlashes?: boolean;

Reply:
keepEmptySegments translate better this option